Triangulation_2/include/CGAL/Constrained_triangulation_2.h

```diff
@@ -101,6 +103,16 @@ class Constrained_triangulation_2
     }
   };
 
+
+  //! Exception thrown for `intersect()` when called with the tag `Throw_on_intersection_tag`.
+  class Is_intersecting_exception : public std::exception
```

Comment:
The class should be updated. I don't  like the name. @lrineau/@afabri (as older maintainers of the package) What do you think of `Intersection_of_constraints_exception`.

Reply:
+ it should be documented